docker/README.md

```diff
@@ -75,3 +75,19 @@ Enter docker container to check any issues
 ```
 docker exec -it <container_id|container_name> /bin/bash
 ```
+
+How to Collect Docker Container Performance Metrics using OMI/OMI provider in Docker Container
+-------
+
+OMI/OMI provider can work on containers, just make sure you know following notes:
+- OMI/OMI provider can collect containers performance metrics in a normal container, but you cannot collect virtual machine host's performance metrics in container.
```

Comment:
Basically, docker provide 3 ways to collect perf: pseudo-files, docker stats command and the Remote API, and the pseudo-files and remote api can collect containers perf in containers, so 2 ways to collect in containers.
but virtual machine host's perf cannot get through these 2 ways in container.
What I got the perf in container is just to run netstat,top.free,df in container which get the perf is different with the perf collected using 2 ways docker provided, but it is same as we directly collect from virutal machine host except tcp connections number.